data/text/abilities.ts

```diff
@@ -0,0 +1,1480 @@
+export const AbilitiesText = {
```

Comment:
Should these files be `data/text/en/abilities.ts`? Or i suppose like with `data/`, only modifications to the default live in subdirs?

Reply:
Yeah, that's probably how I'd end up doing it.